src/test/unit/test_main_utils.py

```diff
@@ -250,8 +254,22 @@ def test_omit_keys(value, output):
     assert omit_keys(value) == output
 
 
+def test_get_omitting_regexes(monkeypatch):
+    monkeypatch.setenv(LUMIGO_SECRET_MASKING_REGEX, '[".*evilPlan.*"]')
+    assert [r.pattern for r in get_omitting_regexes()] == [".*evilPlan.*"]
+
+
+def test_get_omitting_regexes_backward_compatibility(monkeypatch):
+    monkeypatch.setenv(LUMIGO_SECRET_MASKING_REGEX_BACKWARD_COMP, '[".*evilPlan.*"]')
+    assert [r.pattern for r in get_omitting_regexes()] == [".*evilPlan.*"]
+
+
+def test_get_omitting_regexes_fallback(monkeypatch):
+    assert [r.pattern for r in get_omitting_regexes()] == OMITTING_KEYS_REGEXES
+
+
```

Comment:
Add a test for a case that the user has both `LUMIGO_SECRET_MASKING_REGEX` and `LUMIGO_SECRET_MASKING_REGEX_BACKWARD_COMP`